tests/test_readwrite.py

```diff
@@ -831,6 +831,31 @@ def test_adata_in_uns(tmp_path, diskfmt, roundtrip):
     assert_equal(orig, curr)
 
 
+@pytest.mark.parametrize(
+    "uns_val",
+    [
+        pytest.param(dict(base=None), id="dict_val"),
+        pytest.param(
+            pd.DataFrame(dict(col_0=["string", None])),
+            id="df",
+            marks=pytest.mark.xfail(
+                reason="Nullable string arrays not yet implemented"
```

Comment:
Is that true anymore? https://github.com/scverse/anndata/blob/4bcd98936395421c8759e0e30016483786cca2ef/src/anndata/_settings.py#L424-L430